e2e-tests/ukef/cypress/support/trade-finance-manager-ui/submit-deal-cancellation.js

```diff
@@ -0,0 +1,18 @@
+import { TFM_URL } from '../../../../e2e-fixtures';
+import * as caseDealPage from '../../../../tfm/cypress/e2e/pages/caseDealPage';
+import * as checkDetailsPage from '../../../../tfm/cypress/e2e/pages/deal-cancellation/check-details';
+
+export const submitDealCancellation = ({ dealId, effectiveDate = undefined }) => {
```

Comment:
I don't think we need the `effectiveDate` to have a default of undefined - would be undefined if it's not provided 😛